test/controllers/foo_controller_test.rb

```diff
@@ -0,0 +1,3 @@
+begin 
+rescue Exception
```

Comment:
__[staging][rails_best_practices]__
* not rescue Exception [:ref](http://rails-bestpractices.com/posts/2012/11/01/don-t-rescue-exception-rescue-standarderror/)